contracts/CTKNCrowdsale.sol

```diff
@@ -138,8 +177,36 @@ contract CTKNCrowdsale is CappedCrowdsale, RefundableCrowdsale, MintedCrowdsale
      *  @param _weiAmount Value in wei to be converted into tokens
      *  @return the wei amount in excess of what was needed to buy a whole number of tokens.
      */
-    function _getOverpaymentAmount(uint256 _weiAmount) internal view returns (uint256) {
+    function _getOverpaymentAmount(uint256 _weiAmount)
+        internal
```

Comment:
@davesag are you using a certain linter? Want to make sure I use the same one.

Reply:
using `solhint`